### PR TITLE
♻️ refactor: remove build-style.js and tailwind & update storybook to v7.0

### PR DESCRIPTION
**Pull request by AntonyDerache**
# Pull request for Candy-doc components library

## Your checklist for this pull request

🚨 Please review the [guidelines for contributing](../../CONTRIBUTING.md) to
this repository.

- [X] Make sure you are requesting to **pull a feature/bugfix branch** (right
  side). Don't request your master!
- [X] Make sure you are making a pull request against the **main branch** (left
  side).
- [X] Check the commit's or even all commits' message styles matches our
  requested structure.
- [X] Check your code additions will fail neither code linting checks nor unit
  test.

## Description

- Due to not working changes using unocss, it has been removed. As tailwind is not fitting well with the use of lit Elements and is asking workarounds pretty tough, decision has been taken to remove it and make our own css. 
- Storybook update to v7.0 lead to a depreciated type -> refactor all .stories to comply with the new type
- Global code cleaning/refactor in .stories and components
- Navigation and interactions in storybook has been improved 
</br>

❤️ Thank you!

### CODE REVIEW

**Review by Yugo-Cartron — APPROVED** [review]
That's good

**Review by ctacquet — DISMISSED** [review]
Good job !

File: src/components/Alert/Alert.ts [hunk 1]
```diff
@@ -1,26 +1,28 @@
...
+import AlertStyle from "./AlertStyle";
 
 export type CandyAlertProps = {
   type: string;
 };
 
 @customElement("candy-alert")
-export class CandyAlert extends TailwindElement {
+export class CandyAlert extends LitElement {
+  static styles = [AlertStyle];
```
Comment (Yugo-Cartron) [inline src/components/Alert/Alert.ts:13]: static styles = AlertStyle;

**Review by Yugo-Cartron — CHANGES_REQUESTED** [review]
You should keep the same energy bro

**Review by Yugo-Cartron — DISMISSED** [review]
Very good job Anto !

**Review by Yugo-Cartron — APPROVED** [review]

**Pull request merged**